api/optimeet/groups/urls.py

```diff
@@ -10,6 +10,8 @@
     path('<str:group_id>/users/', views.add_users_to_group, name='add_users_to_group'),
     #path('usergroups/', views.UserGroupListAPIView.as_view(), name='usergroup-list'),#get all usergroup
     #path('usergroups/<str:user_id>/', views.UserGroupListAPIView.as_view(), name='usergroup-byuser'),#get usergroup by user_id
-    path("<int:group_id>/recs", views.get_recommendation)
+    path("<int:group_id>/recs", views.get_recommendation),
+    path("<int:group_id>/votes", views.votes, name='votes'),
+    path("<int:group_id>/votescreate", views.create_vote, name='create_vote'),
```

Comment:
Instead of making a new route for getting or creating, we want to leverage HTTP verbs like GET and POST. We can reuse the same `/votes` route and have GET requests go to one function and POST requests go to another.